### PR TITLE
Comments in C_CPP

### PR DESCRIPTION
**Pull request by elixir14082002**
<hr>

## Description 📜

Added Documentation on Comments in C and CPP.
#6389

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by elixir14082002** [comment]
@Suvraneel please review!

File: C_CPP/Fundamentals/Comments in C_CPP/Comments.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+Comments are lines of code which are ignored by the compiler.
+Comments are of 2 types:
+- Single Line Comment
+- Multi Line Comment
+
+### SINGLE LINE COMMENT
+
+Single line comments are used to apply comment to one line only. It is denoted by `// text`. <br>
+<b>SYNTAX:</b>
```
Comment (Suvraneel) [inline C_CPP/Fundamentals/Comments in C_CPP/Comments.md:13]:
```suggestion
**SYNTAX**
```

File: C_CPP/Fundamentals/Comments in C_CPP/Comments.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      cout<<"Hello World"; // Print statement
+      return 0;
+    }
+```
+<b>OUTPUT:</b>
+```
+    Hello World
+```
+
+<b>EXPLANATION:</b>
```
Comment (Suvraneel) [inline C_CPP/Fundamentals/Comments in C_CPP/Comments.md:34]:
```suggestion
**EXPLANATION**
```

File: C_CPP/Fundamentals/Comments in C_CPP/Comments.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+If we take a look at this particular program we can see that the comment lines are used to show the function of certain lines. It is also used to define what the particular code does. As we can see in the above code, the single line comments are used a number of times.
+
+### MULTI LINE COMMENT
+
+Multi line comments are used to apply comment to more than one line only. It is denoted by ` /* text */`. <br>
+<b>SYNTAX:</b>
```
Comment (Suvraneel) [inline C_CPP/Fundamentals/Comments in C_CPP/Comments.md:40]:
```suggestion
Multi line comments are used to apply comment to more than one line only. It is denoted by ` /* text */`.   
**SYNTAX**
```

File: C_CPP/Fundamentals/Comments in C_CPP/Comments.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+```
+    /* Multi Line Comment
+    .
+    .
+    .
+    Comment ends here */
+```
+<b>EXAMPLE:</b>
```
Comment (Suvraneel) [inline C_CPP/Fundamentals/Comments in C_CPP/Comments.md:48]:
```suggestion
**EXAMPLE**
```

File: C_CPP/Fundamentals/Comments in C_CPP/Comments.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+      cout<<"Hello World"; 
+      return 0;
+    }
+```
+<b>OUTPUT:</b>
+```
+    Hello World
+```
+
+<b>EXPLANATION:</b>
```
Comment (Suvraneel) [inline C_CPP/Fundamentals/Comments in C_CPP/Comments.md:66]:
```suggestion
**EXPLANATION**
```

File: C_CPP/Fundamentals/Comments in C_CPP/Comments.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+  #include<stdlib.h>
+
+  int main()
+  {
+    int a=9, b=1; // variable whose sum is to be printed
+    printf("%d", a+b); // sum of the variables
+    return 0;
+  }
+```
+<b>OUTPUT:</b>
```
Comment (Suvraneel) [inline C_CPP/Fundamentals/Comments in C_CPP/Comments.md:84]:
```suggestion
**OUTPUT**
```

**Review by Suvraneel — CHANGES_REQUESTED** [review]

**Comment by Suvraneel** [comment]
fixes #6389

**Comment by Suvraneel** [comment]
@elixir14082002  LGTM ❤️ 🚀

**Pull request merged**